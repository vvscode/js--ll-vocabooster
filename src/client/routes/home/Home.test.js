import React from 'react';
import { shallow } from 'enzyme';
import Component from './Home.component';

describe('Home', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {
      route: {},
    };

    getShallowComponent = () => shallow(<Component {...props} />);
  });

  describe('render', () => {
    it('should render with all props', () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
