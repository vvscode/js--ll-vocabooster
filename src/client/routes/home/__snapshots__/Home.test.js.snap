// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render should render with all props 1`] = `
<Grid
  celled="internally"
>
  <GridRow>
    <GridColumn
      width={5}
    >
      <pre>
        {
  "login": "111",
  "password": "2",
  "text": "",
  "words": []
}
      </pre>
    </GridColumn>
    <GridColumn
      width={11}
    >
      <h4>
        Home
      </h4>
      <Segment>
        <CredentialsForm
          login="111"
          onSubmit={[Function]}
          password="2"
        />
      </Segment>
      <Segment>
        <Connect(TextSourceForm)
          onSubmit={[Function]}
        />
      </Segment>
      <Segment>
        <Connect(TableOfWords)
          onSubmit={[Function]}
          words={Array []}
        />
      </Segment>
      <_class
        footer={null}
        header={null}
        route={Object {}}
      />
       
    </GridColumn>
  </GridRow>
</Grid>
`;
